src/gf_ring.h

```diff
@@ -459,6 +461,61 @@ inline void RingModN<T>::sub_vecp_to_vecp(
     }
 }
 
+/** Butterfly computation for Cooley-Tukey FFT algorithm
+ *
+ * Perform in-place oprations on two buffers `P`, `Q` with a coefficient `c`
+ * \f{eqnarray*}{
```

Comment:
I didn't check the DOxygen output, but don't we need to skip a line before the `\f{eqnarray*}{`?